engineering_tools/vector_map_scripts/create_two_lane_map.py

```diff
@@ -0,0 +1,147 @@
+"""
+This script creates a vector map with two parallel lanes using the Lanelet2 library and saves it as an OSM file.
+The script can be run from the command line with the following arguments:
+- filename: The output filename for the vector map.
+- total_length: The length of the lanes (default is 50.0).
+- lane_width: The width of the lanes (default is 3.7).
+- points_per_lanelet: The number of points in the lane (default is 10).
```

Comment:
I suggest change this to number of points per meter, and do the calculations based on that. Also provide default values for variables in case user misses one.